tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -274,6 +303,982 @@ describe.skip('backbeat routes', () => {
+                assertVersionHasNotBeenUpdated(currentVersion, expectedVersionId);
+
+                const nonCurrentVersion = Versions.find(v => !v.IsLatest);
+                assertVersionIsNullAndUpdated(nonCurrentVersion);
+                return done();
+            });
+        });
+
+        it('should update metadata of a non-version object', done => {
+            let objMD;
+            return async.series([
+                next => s3.putObject({ Bucket: bucket, Key: keyName, Body: new Buffer(testData) }, next),
+                next => makeBackbeatRequest({
+                    method: 'GET',
+                    resourceType: 'metadata',
+                    bucket,
+                    objectKey: keyName,
+                    queryObj: {
+                        versionId: 'null',
+                    },
+                    authCredentials: backbeatAuthCredentials,
+                }, (err, data) => {
+                    if (err) {
+                        return next(err);
+                    }
+                    const { error, result } = updateStorageClass(data, storageClass);
+                    if (error) {
+                        return next(error);
+                    }
+                    objMD = result;
+                    return next();
+                }),
+                next => makeBackbeatRequest({
+                    method: 'PUT',
+                    resourceType: 'metadata',
+                    bucket,
+                    objectKey: keyName,
+                    queryObj: {
+                        versionId: 'null',
+                    },
+                    authCredentials: backbeatAuthCredentials,
+                    requestBody: objMD.getSerialized(),
+                }, next),
+                next => s3.headObject({ Bucket: bucket, Key: keyName }, next),
+                next => s3.listObjectVersions({ Bucket: bucket }, next),
+            ], (err, data) => {
+                if (err) {
+                    return done(err);
+                }
+
+                const headObjectRes = data[3];
+                assert(!headObjectRes.VersionId);
+                assert.strictEqual(headObjectRes.StorageClass, storageClass);
+
+                const listObjectVersionsRes = data[4];
+                const { DeleteMarkers, Versions } = listObjectVersionsRes;
+
+                assert.strictEqual(DeleteMarkers.length, 0);
+                assert.strictEqual(Versions.length, 1);
+
+                const currentVersion = Versions[0];
+                assert(currentVersion.IsLatest);
+                assertVersionIsNullAndUpdated(currentVersion);
+                return done();
+            });
+        });
+
+        it('should create a new null version if versioning suspended and no version', done => {
+            let objMD;
+            return async.series([
+                next => s3.putBucketVersioning({ Bucket: bucket, VersioningConfiguration: { Status: 'Enabled' } },
```

Comment:
You don't need to enable versioning before suspending it, you can just put a versioning-suspended status on a nonversioned bucket (same for the other tests).